README.md

```diff
@@ -6,7 +6,7 @@
 [![Hex.pm](https://img.shields.io/hexpm/dt/pluggy_elixir?style=flat-square)](https://hex.pm/packages/pluggy_elixir)
 [![codecov](https://img.shields.io/codecov/c/github/brainnco/pluggy_elixir?style=flat-square)](https://codecov.io/gh/brainnco/pluggy_elixir)
 
-**TODO: Add description**
+Welcome to Pluggy Elixir, an API client written in Elixir to access the Open Banking services provided by [Pluggy](http://pluggy.ai)
```

Comment:
I'm not sure if we should use the term "Open Banking" yet, because this is not exactly what is doing right now.

We can see details about the Open banking phases here: https://www.bcb.gov.br/estabilidadefinanceira/openbanking